cmd/state-offline-uninstaller/uninstall.go

```diff
@@ -214,13 +213,10 @@ func (r *runner) validateTargetPath(path string) error {
 }
 
 func (r *runner) removeEnvPaths() error {
-	isAdmin, err := osutils.IsAdmin()
-	if err != nil {
-		return errs.Wrap(err, "Could not determine if running as Windows administrator")
-	}
-
 	// remove shell file additions
-	if err := r.shell.CleanUserEnv(r.cfg, sscommon.OfflineInstallID, !isAdmin); err != nil {
+	// Note: on Windows, runtimes are not installed to the Admin %PATH%, even if the user has admin
+	// privileges, so call CleanUserEnv with user scope.
```

Comment:
This doesn't sound right to me. We pass `isAdmin` to `WriteUserEnv()`, and it's respected there.

If `CleanUserEnv()` doesn't handle this input the same as `WriteUserEnv()` then we should address that head-on. The current change feels like we're working around the issue from the consuming code, which is error-prone.

Reply:
The offline installer is not calling `WriteUserEnv()` as far as I can see. I can look into making it do so, but I was under the impression that we were intentionally not running the offline installer in full admin mode because setting a global default as administrator is not supported (https://github.com/ActiveState/cli/blob/d2503b72f9fa33f73b14f2be7c9e97d894cbc088/internal/globaldefault/default.go#L40).

Reply:
After investigation, it was a bug in `ConfigureAvailableShells`. Thanks for suggesting something was amiss.